templates/definition/charger/cfos.yaml

```diff
@@ -2,10 +2,11 @@ template: cfos
 products:
   - brand: cFos
     description:
-      de: Bei cFos PowerBrain charger muss das [`charge meter`](meters) separat konfiguriert werden.
-      en: For cFos PowerBrain charger, the [`charge meter`](meters) must be configured separately.
       generic: PowerBrain
 requirements:
+  description:
+    de: Bei cFos PowerBrain charger muss der cFos Zähler separat als Ladezähler konfiguriert werden [`charge meter`](meters).
```

Comment:
>  Bei cFos PowerBrain charger

Du bist doch schon in der Config des Chargers. Bzw in dessen Doku. Warum muss das nochmal wiederholt werden? Einfach "matter of fact":

> Der Zähler (fall vorhanden) wird nicht automatisch erkannt und muss separat als Ladezähler konfiguriert werden.

Der Link hilft m.E. auch nicht da er während `configure` nicht gültig ist (oder Du setzt einen absoluten Link).

Reply:
Wir wollen ja nicht das Atom spalten sondern nur einen Fakt erwähnen ;)